Example_Programs/codes/VPR.py

```diff
@@ -3,13 +3,13 @@
 
 class vpr(object):
 	def analysis(self,ratinglist):
-		l=len(ratinglist)
-		if (l!=0):
+		len=len(ratinglist)
```

Comment:
len is a python reserved keyword.. so we have to reduce the use of python specific keyword in our code,.. then we can  replace len with length